### PR TITLE
[PLATFORM-236] Chart Module

### PR DESCRIPTION
**Pull request by timoxley**
Implements Chart module via react-highcharts-official.

### CODE REVIEW

**Comment by fonty1** [comment]
The code looks good but I can't seem to get any canvases starting... not sure what's going on here with my environment so I can't be much of a reviewer till I figure what's going on : (

{"code":"CANVAS_UNREACHABLE","message":"Unable to communicate with remote server!"}

File: app/src/editor/shared/components/Subscription.jsx [hunk 1]
```diff
@@ -53,7 +49,7 @@ const ClientProvider = withAuthKey(class ClientProvider extends Component {
 
     teardown() {
         const { client } = this.state
-        if (client) {
+        if (client && client.connection) {
```
Comment (juhah) [inline app/src/editor/shared/components/Subscription.jsx]: This looks like something the `streamr-client` should be checking?

File: app/src/editor/shared/components/modules/Chart.jsx [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+
+import HighchartsReact from 'highcharts-react-official'
+import ModuleSubscription from '$editor/shared/components/ModuleSubscription'
+import SvgIcon from '$shared/components/SvgIcon'
+
+import styles from './Chart.pcss'
+
+const rangeConfig = {
+    All: 'all',
+    '1 month': 30 * 24 * 60 * 60 * 1000,
```
Comment (juhah) [inline app/src/editor/shared/components/modules/Chart.jsx]: I think you'll need to separate the keys and the labels for translation.

Comment (timoxley) [reply app/src/editor/shared/components/modules/Chart.jsx]: Ah yeah you're right.

File: app/src/editor/shared/components/Subscription.jsx (hunk 1 above)
Comment (timoxley) [reply app/src/editor/shared/components/Subscription.jsx]: Yeah agreed

**Comment by timoxley** [comment]
> Resizing the module once a chart is shown is a bit wonky though.

@juhah yes, I believe this gets fixed later on after removing react-dnd.

**Comment by timoxley** [comment]
> I can't seem to get any canvases starting

@fonty1 you might need to update your e&e images or `git pull` in `streamr-docker-dev`?

This is how I start e&e with the latest images:
```
streamr-docker-dev bind-ip; docker pull streamr/engine-and-editor:dev && docker pull streamr/ethereum-watcher:latest; docker pull streamr/cloud-broker:stg; docker pull streamr/data-api:latest; streamr-docker-dev start 1 && sleep 60 && streamr-docker-dev start ethereum-watcher
```

**Comment by timoxley** [comment]
@fonty1 to get this stack of PRs moving forward I'm going to merge this but I'd appreciate it if you try this out and me know if you hit any issues.

**Pull request merged**